app/styles/component/_site-head.scss

```diff
@@ -149,3 +157,32 @@ body:not(.no-scroll) {
 		}
 	}
 }
+
+.site-head__partner-slot {
+	background: white;
+	border-bottom: 1px solid $vapor;
+	height: $partner-slot-height;
+	position: fixed;
+	top: 55px;
+	width: 100%;
+	z-index: 695;
+
+	.featured-video-on-scroll-active & {
+		position: static;
```

Comment:
we should apply `position: static` also if `bfaa-template` class is applied, see L148 and L156